### ISSUE TITLE
Network Load Balancer now supports UDP protocol

### ISSUE THREAD

**Issue by ewbankkit** [terraform-providers/terraform-provider-aws#9109]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Elastic Load Balancing now supports the UDP protocol on Network Load Balancers.

### New or Affected Resource(s)

* [`aws_lb_target_group`](https://www.terraform.io/docs/providers/aws/r/lb_target_group.html)

### Potential Terraform Configuration

```hcl
resource "aws_lb_target_group" "example" {
  protocol = "UDP"
}
```

### References

* [Announcement](https://aws.amazon.com/about-aws/whats-new/2019/06/network-load-balancer-now-supports-udp-protocol/)
* [Blog post](https://aws.amazon.com/blogs/aws/new-udp-load-balancing-for-network-load-balancer/)

**Comment by stack72** [comment]
This requires an update to SDK version 1.20.7

**Comment by stack72** [comment]
https://github.com/terraform-providers/terraform-provider-aws/pull/9084

**Comment by claydanford** [comment]
Hello,

I am interested in contributing to this, but need some guidence as this is my first time working on this project.

I was able to make these changes, adding UDP, and TCP_UDP in the lb_listener resource and lb_target_group resource and deploy them.

What other items should I look at if I wanted to contribute these changes?

Thank you,

CRD

**Comment by bflad** [comment]
Support for `TCP_UDP` and `UDP` in the `protocol` for the `aws_lb_listener` and `aws_lb_target_group` resources has been merged and will release with version 2.17.0 of the Terraform AWS Provider, later this week. 👍 Thanks, @stack72!

**Comment by bflad** [comment]
This has been released in [version 2.17.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2170-june-28-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

For further feature requests or bug reports with this functionality, please create a [new GitHub issue](https://github.com/terraform-providers/terraform-provider-aws/issues/new/choose) following the template for triage. Thanks!

**Comment by sadoha** [comment]
Thank you for update. Everything works well in a TCP_UDP mode.
